tests/pool_test.go

```diff
@@ -622,23 +622,17 @@ func newPoolFromVMI(vmi *v1.VirtualMachineInstance) *poolv1.VirtualMachinePool {
 	return pool
 }
 
-func notDeletedVMs(poolName string, vms *v1.VirtualMachineList) (notDeleted []v1.VirtualMachine) {
-	nonDeletedVms := NotDeletedVMs(vms)
-	for _, vm := range nonDeletedVms {
+func filterNotDeletedVMIsOwnedByPool(poolName string, vms *v1.VirtualMachineList) []v1.VirtualMachine {
```

Comment:
The function operates on VMs:
```suggestion
func filterNotDeletedVMsOwnedByPool(poolName string, vms *v1.VirtualMachineList) []v1.VirtualMachine {
```

Reply:
Done, thanks.